### PR TITLE
add dutch translation

### PR DESCRIPTION
**Pull request by m11tch**
add dutch language

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -364,5 +375,8 @@
       "tags": [],
       "implicitDependencies": ["ui"]
     }
+  },
+  "cli": {
+    "analytics": false
```
Comment (dtslvr) [inline angular.json:380]: Why?

File: apps/client/src/locales/messages.nl.xlf [hunk 2]
```diff
@@ -0,0 +1,2650 @@
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" target-language="nl" datatype="plaintext" original="ng2.template">
+    <body>
+      <trans-unit id="e19fcf996343543e13789f17b550ab0c08124b5c" datatype="html">
+        <source>Create Account</source>
+		<target state="translated">Account aanmaken</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.nl.xlf:6]: Can you please replace the tabs with two spaces that it matches the style?

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution @m11tch. That already looks quite good, also with the `angular.json` setup 😃 

There are some more changes needed, see [here](https://github.com/ghostfolio/ghostfolio/pull/1276/files):

* `apps/api/src/app/frontend.middleware.ts`
* `apps/client/src/app/pages/account/account-page.component.ts`
* `libs/common/src/lib/helper.ts`

Could you add the remaining pieces?

File: angular.json (hunk 1 above)
Comment (m11tch) [reply angular.json]: Apologies, seems VSCode added this automatically.. removed this again.

File: apps/client/src/locales/messages.nl.xlf (hunk 2 above)
Comment (m11tch) [reply apps/client/src/locales/messages.nl.xlf]: fixed.

**Review by dtslvr — APPROVED** [review]
That looks fine. Great work and thanks for your changes! 😎

**Pull request merged**